### DISCUSSION TITLE
Whitelist and Blacklist

### DISCUSSION THREAD

**Discussion by Sidetrack-CA** [prasathmani/tinyfilemanager#545] [Q&A] [answered]
Is there a way to whitelist or blacklist an entire subnet of a network?

Example: 

Whitelist: 192.168.1.0/24 (Entire 192.168.1.0-254)
blacklist: 0.0.0.0 (blacklist all other IP addresses)

thanks for making the best file manager I have found and works so well.

Rob

**Comment by prasathmani** [comment, marked as answer]
configure `$ip_ruleset = 'ON';` and for wildcard whitelist you need modify the code, check this out https://stackoverflow.com/a/35559544/1164642
 
![image](https://user-images.githubusercontent.com/11345598/118342290-5eec2200-b540-11eb-81d9-65f6d0e2501b.png)

**Reply by Sidetrack-CA** [reply]
Thank you so much for the answer.  This is really a great little file manager.  I love how it works and its super easy to use.
